src/CreateSignalFactory.js

```diff
@@ -100,6 +113,7 @@ module.exports = function (controller, model, services, compute, modules) {
             signal.isExecuting = false
             controller.emit('signalEnd', {signal: signal})
             controller.emit('change', {signal: signal})
+            emitControllerStatus(-1)
```

Comment:
Is it possible that signal fail before reaching here and it will not be executed?